src/Main.java

```diff
@@ -1,10 +1,21 @@
 import java.util.Scanner;
-public class Main{
-    int Year = 2023;
-
-    public static int MYMETHOD(int a, int b) {
+public class Main {
+    public static int sumOfNumbers(int a, int b) {
         int c = a + b;
-        return c;}
+        System.out.println("Result:\n"+c);
+        return 0;
```

Comment:
This variant is also correct but more logically it is not to write 'return 0' and mark our method as 'void' instead of 'int'